data/scripts/debug.inc

```diff
@@ -338,6 +323,27 @@ Debug_EventScript_InflictStatus1_Single:
 	releaseall
 	end
 
+Debug_EventScript_InflictStatus1_Party:
```

Comment:
I forgot to mention this earlier: can you move this section back to where it was?